vms/administrator/tests/test_formFields.py

```diff
@@ -188,8 +185,8 @@ def test_null_values_in_create_shift(self):
         created_event = create_event_with_details(event)
 
         # create job with values
-        job = ['job', '2017-08-21', '2017-08-21', '',created_event]
-        created_job = create_job_with_details(job)
+        job = ['job', '2017-08-21', '2017-08-21', '', created_event]
+        create_job_with_details(job)
```

Comment:
we need to assign this to ```created_job```.

Reply:
I will udate all the warnings in other pull request, these assignment statements got changed when i ran autoflake commands. But i will take care of these in other PR.

Reply:
Cool!